metaplex-foundation/metaplex
# feat: partially opened pack redemption

## Description

Implements partially opened pack redemption. Works the same way as for unopened pack redeem.

## References
https://app.asana.com/0/1201289799176936/1201477388604513/f

## Testing

1. Go to the unopened pack page
2. Click on "Open pack"
3. Approve the first transaction
4. Wait for the second transaction
5. Cancel the second transaction.
6. Redirect to My Items
7. Open that pack page once more
8. Click on "Resume pack opening"
9. Confirm the second transaction.
10. Observe Pack opening modal closes.
11. Go to my items
12. Observe redeemed items and pack status changed to "0 Cards to Redeem"
